### PR TITLE
Accordion added to sidebar

### PR DESCRIPTION
**Pull request by Era-cell**
This PR to be taken post [dropdown ](https://github.com/Intelligence-AI/learnjsonschema.com/pull/165)toggle after its moved to breadcrumb.
The issues I am facing are:
1.Moving css to assets/main.scss isn't applying any CSS on the partials/layouts/sidebar.html.
2.What should the switching be named. "Toggle vocab/alphabetical"
   or should we show full information on toggle??

### CODE REVIEW

**Comment by jviotti** [comment]
Your PR is trying to do a lot of things at once:

- Import and setup the dropdown
- Add accordions
- Add alphabetic sorting to the sidebar

Please do only one thing per PR. In this case, this should only be adding the accordions, without doing or importing anything else!

**Comment by jviotti** [comment]
In any case, I don't think we need the alphabetic ordering and the toggle. Just an accordion is more than enough.

**Comment by Era-cell** [comment]
> In any case, I don't think we need the alphabetic ordering and the toggle. Just an accordion is more than enough.

I should have mentioned this: but alphabetical ordering is from this issue requirement:
https://github.com/Intelligence-AI/learnjsonschema.com/issues/36

**Comment by jviotti** [comment]
We can do both, but as separate PRs. Otherwise its hard to review and merge a PR that does multiple things at once

**Comment by jviotti** [comment]
@Era-cell I've been making some changes to the site and this PR now has conflicts. Please let me know when it is on a reviewable state!

File: layouts/partials/sidebar.html [hunk 1]
```diff
@@ -1,37 +1,42 @@
 <aside id="sidebar" class="border-end position-relative d-none d-xl-block">
-  <div class="px-4 py-2 top-0 start-0 bottom-0">
+  
+  <div class="px-4 py-2">
```
Comment (jviotti) [inline layouts/partials/sidebar.html:3]: Let's remove the blank lines around this line

File: layouts/partials/sidebar.html [hunk 2]
```diff
@@ -1,37 +1,42 @@
...
-                      {{ if eq $.Permalink .Permalink }}
-                        <a class="text-muted fw-bold text-decoration-none font-monospace active"
-                          href="{{ ref $ $keywordURL }}" aria-current="true">{{ .Params.keyword }}</a>
-                      {{ else }}
-                        <a class="text-body-tertiary text-decoration-none font-monospace"
-                          href="{{ ref $ $keywordURL }}">{{ .Params.keyword }}</a>
-                      {{ end }}
-                    </li>
-                  {{ end }}
+              {{ if eq .Section $dialect }}
```
Comment (jviotti) [inline layouts/partials/sidebar.html:21]: Can you keep the indentation? For some reason this entire block moved one tab to the left

File: layouts/partials/sidebar.html [hunk 3]
```diff
@@ -1,37 +1,42 @@
...
+              {{ if eq .Parent.Title $vocabulary }}
```
Comment (jviotti) [inline layouts/partials/sidebar.html:22]: Also here, the indentation here is completely broken

File: layouts/partials/sidebar.html [hunk 4]
```diff
@@ -1,37 +1,42 @@
...
-            {{ $vocabularyURL := (replace $vocabulary " " "-") | lower | printf "%s/%s" $dialectURL }}
-            <a class="ps-0 nav-link text-body fw-bold" href="{{ ref $ $vocabularyURL }}">
-              <i class="bi bi-chevron-down"></i> {{ $vocabulary }}
-            </a>
+        <li class="nav-item">
+          {{ $vocabulary := .Title }}
+          {{ $vocabularyURL := (replace $vocabulary " " "-") | lower | printf "%s/%s" $dialectURL }}
+          <details open>
+            <summary class="ps-0 nav-link text-body">
+              <i class="rotate bi bi-chevron-down"></i>
```
Comment (jviotti) [inline layouts/partials/sidebar.html:16]: Is this `rotate` class used anywhere? Can you check there are no unused classes in any of these changes?

File: assets/main.scss [hunk 5]
```diff
@@ -129,3 +129,8 @@ article table {
 .code-instance + .code-schema {
   @extend .mt-4;
 }
+
+details:not([open]) summary i {
+  display: inline-block;
```
Comment (jviotti) [inline assets/main.scss:134]: Please do this using the Bootstrap helpers:

```scss
@extend .d-inline-block;
```

**Comment by Era-cell** [comment]
Hi, @jviotti you can review the PR now...

**Review by jviotti — APPROVED** [review]
Awesome!

**Pull request merged**